web-ui/src/components/member_selector/MemberSelector.jsx

```diff
@@ -4,30 +4,20 @@ import {
   Avatar,
   Card,
   CardHeader,
-  Collapse,
   Divider,
   IconButton,
   List,
-  ListItem,
-  ListItemAvatar,
   ListItemIcon,
   ListItemText,
-  Menu,
-  MenuItem,
   Tooltip,
   Typography
 } from '@mui/material';
-import AddIcon from '@mui/icons-material/Add';
```

Comment:
Some of these icons are no longer used and access to the ones that are was simplified by using their given names.